pkg/schema_store/utils.go

```diff
@@ -178,7 +178,10 @@ func GetTableDefFromDB(db *sql.DB, dbName string, tableName string) (*Table, err
 		} else {
 			column.IsPrimaryKey = false
 		}
-		if extra.Valid && strings.Contains(strings.ToUpper(extra.String), "GENERATED") {
+
+		// TiDB describes certain column as `DEFAULT_GENERATED`
+		if extra.Valid && (strings.Contains(strings.ToUpper(extra.String), "VIRTUAL GENERATED") ||
```

Comment:
better to cache the result of "IsTidb" , and use that to identiry tidb.

Reply:
`IsTiDB` does not help here. 
It distinguishes whether a column is generated or not, not TiDB or not.